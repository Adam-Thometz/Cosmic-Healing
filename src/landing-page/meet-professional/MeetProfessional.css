.MeetProfessional {
  display: flex;
  justify-content: space-between;
  align-items: center;
  margin: 5% 0;
}

.MeetProfessional-img-wrapper {
  width: 40%;
}

.MeetProfessional-img-wrapper img {
  width: 100%;
}

.MeetProfessional-text-wrapper {
  width: 40%;
  padding: 5%;
  margin: 0 auto;
  text-align: left;
}

.MeetProfessional-text-wrapper article {
  font-family: Nuckle;
  line-height: 27px;
}

.MeetProfessional button {
  margin: 32px 0;
}

@media (max-width: 600px) {
  .MeetProfessional {
    flex-direction: column;
  }

  .MeetProfessional-text-wrapper {
    width: 80%;
  }
}