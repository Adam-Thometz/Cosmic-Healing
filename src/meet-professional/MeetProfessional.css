.MeetProfessional {
  display: flex;
  justify-content: space-between;
  align-items: center;
}

.MeetProfessional-img-wrapper {
  width: 40%;
}

.MeetProfessional-img-wrapper img {
  width: 100%;
}

.MeetProfessional-text-wrapper {
  width: 35%;
  margin: 0 auto;
  text-align: left;
}

.MeetProfessional-text-wrapper article {
  font-family: Nuckle;
  line-height: 27px;
}

.MeetProfessional button {
  margin: 32px 0;
}