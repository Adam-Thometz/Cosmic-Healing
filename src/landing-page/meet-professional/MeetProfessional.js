import React from 'react';

import './MeetProfessional.css';

import Link from '../../_components/link/Link';
import SmallCaption from '../../_components/small-caption/SmallCaption';
import SectionHeader from '../../_components/section-header/SectionHeader';

import miriam from '../../_media/miriam2.png';
import stars from '../../_media/stars.png';

const MeetProfessional = ({ style }) => {
  return (
    <section className='MeetProfessional' style={style}>
      <aside className='MeetProfessional-img-wrapper'>
        <img src={miriam} alt='' />
      </aside>
      <aside className='MeetProfessional-text-wrapper'>
        <img className='MeetProfessional-stars' src={stars} alt='' />
        <SmallCaption text="MEET THE LICENSED PROFESSIONAL" />
        <SectionHeader text="Hello, my name is Miriam!" />
        <article>I offer sessions designed to help you shift subconscious imprinting, uncover your authentic self and align with your purpose. As a licensed social worker and spiritual life coach, I use a critically conscious lens and heart-centered approach to help cultivate inner peace and calm. Letâ€™s go on this journey together as we sort through and make sense of each puzzle piece to form a full picture.</article>
        <Link secondary ariaLabel='Click here to learn more about Miriam'>Read more</Link>
      </aside>
    </section>
  );
};

export default MeetProfessional;