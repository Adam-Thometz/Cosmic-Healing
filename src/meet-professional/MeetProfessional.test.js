import React from "react";
import { render } from "@testing-library/react";
import MeetProfessional from './MeetProfessional';

describe('MeetProfessional component', () => {
  it('renders without crashing', () => {
    render(<MeetProfessional />);
  });

  it('matches the snapshot', () => {
    const { asFragment } = render(<MeetProfessional />);
    expect(asFragment()).toMatchSnapshot();
  });
});