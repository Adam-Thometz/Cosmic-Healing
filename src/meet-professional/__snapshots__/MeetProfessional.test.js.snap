// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MeetProfessional component matches the snapshot 1`] = `
<DocumentFragment>
  <section
    class="MeetProfessional"
  >
    <aside
      class="MeetProfessional-img-wrapper"
    >
      <img
        alt=""
        src="miriam2.png"
      />
    </aside>
    <aside
      class="MeetProfessional-text-wrapper"
    >
      <small
        class="SmallCaption"
      >
        MEET THE LICENSED PROFESSIONAL
      </small>
      <h2
        class="SectionHeader"
      >
        Hello, my name is Miriam!
      </h2>
      <article>
        I am a licensed social worker and spiritual life coach who strives to help you uncover your purpose. I use a critically conscious lens, heart-centered approach and astrology charts to align with your higher authentic self. Let's go on this journey together as we sort through and make sense of each puzzle piece to form a full picture.
      </article>
      <button
        class="Button tertiary"
      >
        Read more
        <img
          alt=""
          class="Button-arrow"
          src="arrow.png"
        />
      </button>
    </aside>
  </section>
</DocumentFragment>
`;
